packages/teleport/src/components/RecoveryCodes/RecoveryCodes.tsx

```diff
@@ -40,6 +42,14 @@ export default function RecoveryCodesDialog({
     });
   };
 
+  const handleContinue = () => {
+    userEventService.capturePreUserEvent({
+      event: CaptureEvent.PreUserRecoveryCodesContinueClickEvent,
+      username: username,
+    });
+    onContinue();
```

Comment:
should we put a comment that says that this event is only successfully captured with reset/invite for the moment? it looks like if we don't send in the username with the `capturePreUserEvent` request it fails silently to capture